src/schema/basic_slots.yaml

```diff
@@ -890,6 +890,13 @@ enums:
       LC-DDA-MS/MS Raw Data:
         description: Liquid chromatographically separated MS1 and Data-Dependent MS2 binary instrument file
 
+      Configuration toml:
+        description: >-
+          Configuration toml file used by CoreMS in the NomAnlaysis and MetabolomicsAnalysis workflows to 
```

Comment:
- I don't thin CoreMS is defined anywhere in the schema. At a minimum, please provide a good see also
- please double check that the schema elements that you mention can be looked up verbatim in the docuemtnation or schema files
    - NomAnlaysis -> NomAnalysis